src/constants/endpoints.ts

```diff
@@ -41,10 +34,10 @@ export type DocumentsType = DocumentType[]
 
 export const ENDPOINTS = {
   professions: {
-    all: '/disciplines'
+    all: '/disciplines_by_level'
   },
   subCategories: {
-    all: '/training_set/:id'
+    all: '/training_set'
```

Comment:
Why is there no `id` anymore here?

I am really confused with the naming here as well, probably nothing for this PR though. I think we should be consistent. +1 for a glossary and consistent naming :D

Reply:
In useLoadDisicplines we append the id, but just appending it is easier, than searching for `:id` and replace it, isn't it?